style.css

```diff
@@ -797,12 +794,257 @@ nav {
     font-weight: bolder;
     font-size: 32px;
   }
-
-  .minMenu {
-    display: none;
-  }
 }
 
 .mainMenu {
   display: none;
 }
+
+/* popup styles */
+.modal {
+  display: none;
+  pointer-events: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background: #c1c7d0;
+  opacity: 0.9;
+}
+
+.modal-content {
+  width: 95%;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: #fff;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.modal-header h2 {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+}
+
+.modal-close {
+  cursor: pointer;
+  pointer-events: auto;
+}
+
+.popUpList {
+  display: flex;
+}
+
+.popUpList li:first-child {
+  color: #344563;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+#modal0 .modal-content .modal-content-placeholder {
+  width: 100%;
+  height: 30vh;
+  margin: 10px 0;
+  border-radius: 8px;
+  background: url("/images/Snapshoot_Portfolio.png") no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+#modal1 .modal-content .modal-content-placeholder {
+  width: 100%;
+  height: 30vh;
+  margin: 10px 0;
+  border-radius: 8px;
+  background: url("./images/Snapshoot_Portfolio\(3\).png") no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+#modal2 .modal-content .modal-content-placeholder {
```

Comment:
- [ ] The image width and height of the popup modal do not match that of the Figma template for the desktop version. Kindly make sure it matches it as much as possible. I will suggest you reduce the width of the modal, and add to the height of the image.

Your design
![Capture](https://user-images.githubusercontent.com/85095702/163380049-9811088c-35bb-4015-9b9a-0b108c9dee35.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/163380110-1bd0ce93-dec9-4c65-9c36-b0fe84a3eb09.PNG)